src/main/resources/test_files/runtime/TypeChecking/LetClauseType2.jq

```diff
@@ -0,0 +1,3 @@
+(:JIQS: ShouldCrash; ErrorCode="XPTY0004"; ErrorMetadata="LINE:2:COLUMN:22:" :)
+let $y as integer := (1, 2)
+return $y
```

Comment:
This file should be named as 'LetClauseTypeCheckError[number].jq'

Reply:
DONE